### DISCUSSION TITLE
how to hide method with loopback 4

### DISCUSSION THREAD

**Discussion by emazzu** [loopbackio/loopback-next#7318] [Q&A] [answered]
Hi people, 

before I did 
var isStatic = true;
MyModel.disableRemoteMethod('deleteById', isStatic);

with loopback 4, I dont know, could you help me ?

Regards

**Comment by marioestradarosa** [comment, marked as answer]
Read the following 

https://loopback.io/doc/en/lb4/migration-models-methods.html

In general just remove it from the controller.

**Reply by emazzu** [reply]
@marioestradarosa 
Thank you very much Mario.
I found it.
